argilla/docs/how_to_guides/export.md

```diff
@@ -0,0 +1,156 @@
+---
+description: In this section, we will provide a step-by-step guide to show how to filter and query a dataset.
+---
+
+# Exporting from Argilla
```

Comment:
Perhaps we can add more explicit naming?
```suggestion
# Importing and exporting datasets and records
```

Reply:
not that this would also need to be changed in other places like the `index.md` and the `mkdocs.yaml`